apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -103,6 +118,7 @@ export default function SectionsSetUpContainer({sectionToBeEdited}) {
   const [advancedSettingsOpen, setAdvancedSettingsOpen] = useState(false);
 
   const isNewSection = !sectionToBeEdited;
+  const initialSection = useRef(sectionToBeEdited);
```

Comment:
👀 i don't understand how `useRef` works yet, why do we want this here?

Reply:
It allows us to store an 'initial' that doesn't update every rerender. In the editSectionForm, we keep track of initial values for the section we're editing, but we hadn't done that here. My initial approach was to use lodash .clone(), but Bethany pointed out that it would change every render. With useRef, I can store the value once and access it later without that hassle.